### PR TITLE
Patch2

### PR DESCRIPTION
**Pull request by Saha-7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1766"></a>

### CODE REVIEW

**Comment by Saha-7** [comment]
First of all sorry for that mistakes. I have changed my mistakes.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/PritamSaha7.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Pritam Saha",
+    "type": "personal",
+    "bio": "I am pritam Saha, Computer Science graduate from Adamas University",
```
Comment (Panquesito7) [inline public/data/PritamSaha7.json:4]: You might want to capitalize your name here.
```suggestion
    "bio": "I am Pritam Saha, Computer Science graduate from Adamas University",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Comment (Saha-7) [reply public/data/PritamSaha7.json:4]: I have changed the name to capital letter. Is it visible ? please let me know. 
Or should I have to do it from scratch again ?

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks. 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

**Pull request merged**